### PR TITLE
[Tests-Only] used small skeleton dirs more wisely on apiAuth feature tests

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- skeleton file/folders are used more wisely on the following features suite
  - apiAuth
  - apiAuthOcs
  - apiAuthWebDav

## Related Issue

- Part of https://github.com/owncloud/QA/issues/658

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature [hunk 1]
```diff
@@ -2,12 +2,7 @@
 Feature: delete file/folder
 
   Background:
-    Given user "Alice" has been created with default attributes and without skeleton files
-    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"
-    And user "Alice" has uploaded file with content "some data" to "/textfile1.txt"
-    And user "Alice" has created folder "/PARENT"
-    And user "Alice" has created folder "/FOLDER"
-    And user "Alice" has uploaded file with content "some data" to "/PARENT/parent.txt"
+    Given user "Alice" has been created with default attributes and small skeleton files
```
Comment (Talank) [inline tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature:5]: is it really wise to use small skeleton instead of without skeleton? 🤔

**Review by dpakach — COMMENTED** [review]
lgtm :+1:

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature:5]: no its not :smile:

Comment (kiranparajuli589) [reply tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature]: fixed

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature [hunk 2]
```diff
@@ -2,11 +2,7 @@
 Feature: create folder using MKCOL
 
   Background:
-    Given user "Alice" has been created with default attributes and without skeleton files
-    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"
-    And user "Alice" has created folder "/PARENT"
-    And user "Alice" has created folder "/FOLDER"
-    And user "Alice" has uploaded file with content "some data" to "/PARENT/parent.txt"
+    Given user "Alice" has been created with default attributes and small skeleton files
```
Comment (individual-it) [inline tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature:5]: why changing this back if it was already using "without" skeleton?

**Review by individual-it — COMMENTED** [review]
just one question, I'm confused why we had to change that back

File: tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/apiAuthWebDav/webDavMKCOLAuth.feature]: this PR was before the detailed discussion...i'd to revert this.

**Comment by kiranparajuli589** [comment]
> just one question, I'm confused why we had to change that back

reverted back!

**Review by individual-it — APPROVED** [review]

**Pull request merged**